src/main/java/org/codehaus/mojo/taglist/TagListReport.java

```diff
@@ -372,12 +392,19 @@ private String getRelativePath(File location) {
      *
      * @see org.apache.maven.reporting.MavenReport#canGenerateReport()
      */
+    @Override
     public boolean canGenerateReport() {
         boolean canGenerate = !getSourceDirs().isEmpty();
         if (aggregate && !getProject().isExecutionRoot()) {
             canGenerate = false;
         }
-        return canGenerate;
+
+        if (canGenerate) {
+            executeAnalizes();
```

Comment:
Is this supposed to be https://www.oxfordlearnersdictionaries.com/definition/english/analysis?q=analysis?

Reply:
I follow existing name for `FileAnalyser` class - should be also changed?

Reply:
method renamed

Reply:
> I follow existing name for `FileAnalyser` class - should be also changed?

Needs a separate issue